spec/controllers/admin/works_controller_spec.rb

```diff
@@ -122,19 +122,18 @@
   end
 
   context "#batch_publish_toggle", logged_in_user: :admin do
```

Comment:
Without the :tiff trait, the asset actually ends up with a png file, which it is now illegal to publish.